cmd/validate/schemas/adhoc_schema.json

```diff
@@ -0,0 +1,103 @@
+{
```

Comment:
are these taken from `https://github.com/Yelp/paasta/blob/master/paasta_tools/api/api_docs/swagger.json`?

if yes, i would prefer if we copied the whole file and had a test that checks if it's out of sync with content from the url above.